packages/components/src/components/bal-file-upload/bal-file-upload.tsx

```diff
@@ -16,10 +16,16 @@ export class FileUpload {
   @State() isOver = false
   @State() files: File[] = []
 
+  @Watch('value')
+  onValueChange() {
+    this.files = this.value
```

Comment:
Could we verify that the files and value have changed?
And then only update the files when the value is different.

We have a [areArraysEqual](https://github.com/baloise/web-app-utils/blob/master/packages/utils/README.md#arearraysequal) function in the utils.

Reply:
@sladomic updated

Reply:
@hirsch88 done